ch_tools/monrun_checks/dns.py

```diff
@@ -29,30 +29,35 @@ def __init__(self, fqdn: str, private: bool, strict: bool):
         self.strict = strict
 
 
-@click.command("ext_ip_dns")
```

Comment:
Renamed the command from `ext_ip_dns` to `dns` cause:
- `ext`prefix is incorrect as the command checks not only external IPs;
- The command name doesn't conform naming style of other commands - it's used `_` instead of `-` as a word separator.